src/client/todos/todo.styl

```diff
@@ -0,0 +1,26 @@
+.todo-item
+  font-size .875em
+  margin-bottom .25em
+  max-width 42em
+
+  &__label, &__button
```

Comment:
Ok, so to throw more food for thoughts regarding CSS best practises. OOCSS, SMACSS, and BEM - https://www.youtube.com/watch?v=IKFq2cSbQ4Q
OOCSS is kind of reflection how object orientated CSS could be structured, which is good for us as mainly JS devs ) But it makes sense in larger projects and probably out of scope for Este